### PR TITLE
#137 Changed tr1::unordered_map to boost::unordered_map.

### PR DESCRIPTION
**Pull request by gautierbureau**

### CODE REVIEW

File: dynawo/sources/Modeler/Common/DYNSubModel.cpp [hunk 1]
```diff
@@ -51,7 +51,7 @@ using std::ofstream;
 using std::stringstream;
 using std::vector;
 using std::map;
-using std::tr1::unordered_map;
+using boost::unordered_map;
```
Comment (rosiereflo) [inline dynawo/sources/Modeler/Common/DYNSubModel.cpp]: You also need to remove the #include <tr1/unordered_map> in all files

Comment (gautierbureau) [reply dynawo/sources/Modeler/Common/DYNSubModel.cpp]: It is done as far as I saw mayber I missed one

**Comment by gautierbureau** [comment]
> Are you sure this is going to pass on grovsla6?

Yes boost 1.59 integrates unordered_map

File: dynawo/sources/Modeler/Common/DYNSubModel.cpp (hunk 1 above)
Comment (rosiereflo) [reply dynawo/sources/Modeler/Common/DYNSubModel.cpp]: Yes saw it

**Pull request merged**